File: problem_thirty_three.py
inp = int(input("Enter divi 1: "))
inp2 = int(input("Enter divi 2 :"))
inp3 = int(input("Enter value:"))
def MyFunction(div1,div2,value):
    for i in range(1,value,1):
        mod = i % div1
        mod1 = i % div2
        if mod ==0 and mod1==0:
            print(i)
MyFunction(inp,inp2,inp3)

File: problem_thirty_three.js

var inp = 5
var inp1 = 6
var inp2 = 100
function MyFunction(div1,div2,value){
    for(var i = 1; i < value; i++ ){
        var mod1 = i % div1
        var mod2 = i % div2
        if (mod1 == 0 && mod2==0){
            console.log(i)
        }
    }
}
MyFunction(inp,inp1,inp2)